projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -26,122 +26,124 @@
 </div>
 <ng-container *ngIf="record">
   <!-- HEADER -->
-  <header class="col-10 mt-3 mb-3">
-    <!-- TYPE -->
-    <ng-container *ngIf="record.metadata.type">
-      <span id="doc-type">{{ record.metadata.type | translate | titlecase }}</span>
-    </ng-container>
+  <header class="col-10">
+    <div class="row mt-3 mb-3">
+      <div class="col-2">
+        <shared-thumbnail [record]="record"></shared-thumbnail>
+      </div>
+      <div class="col">
+      <!-- TITLE -->
+      <h2 id="doc-title" *ngIf="record.metadata.title && record.metadata.title | mainTitle as title" class="pl-0">
+        <ng-core-text-read-more
+          [text]="title"
+          [unit]="'character'"
+          [limit]="150"
+          [showMoreLabel]="'Show more' | translate"
+          [showLessLabel]="'Show less' | translate"
+        ></ng-core-text-read-more>
+      </h2>
```

Comment:
indentation problem. For very large file, a good indentation is a "must-have"

Reply:
Done